README.md

```diff
@@ -1,28 +1,25 @@
-# `kcp` is a minimal Kubernetes API server
+# `kcp` is a control plane for applications running across multiple Kubernetes clusters
```

Comment:
I just saw a very interesting and informative presentation from @sttts , and got a little different take-away.  The message I got is that kcp is first and foremost about hosting many workspaces, and secondarily the "workload service" (something that schedules workload described in a kcp workspace onto some set of compute clusters --- if I understood correctly) is expected to be the killer app that draws in usage.  If this split is really right, I think the README should make it really clear.  I myself am working on problems that are much more concerned with control plane issues than scheduling compute.

Reply:
The slogan here now might be misunderstood as federation. Maybe a better term is "... for applications whose workloads are distributed over multiple attache Kubernetes clusters" or something like that. Distributed rather than running.

Reply:
Thanks for your feedback! This wording was a suggestion from Clayton. I'm going to make some minor edits, and future revisions can come in future PRs